src/style.css

```diff
@@ -1 +1,87 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+:root {
+  --text-font: Avenir, sans-serif;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.body {
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  margin-top: 0;
+  margin-bottom: 0;
```

Comment:
According to the checklist, be consistent with your margins. It's recommended to add only top or bottom margin, but not both to avoid potential margin collapse issues.